### PR TITLE
Play with bot

### PR DESCRIPTION
**Pull request by mattmoor**

### CODE REVIEW

**Comment by mattmoor-sockpuppet** [comment]
PR event: opened

**Comment by mattmoor** [comment]
Hello there.

**Comment by mattmoor-sockpuppet** [comment]
Hello @mattmoor

**Comment by mattmoor-sockpuppet** [comment]
PR event: closed

**Comment by mattmoor-sockpuppet** [comment]
PR event: reopened

**Comment by mattmoor-sockpuppet** [comment]
PR event: synchronize

**Comment by mattmoor-sockpuppet** [comment]
PR event: labeled

**Comment by mattmoor-sockpuppet** [comment]
PR event: unlabeled

File: README.md [hunk 1]
```diff
@@ -119,3 +121,6 @@ baz/
 * Handle layer limits
 * Handle symlinks
 * Heuristics around whether to reuse the previous image at all
+BLAH
+BLAH
```
Comment (mattmoor) [inline README.md:125]: What about:
```suggestion
BLURG
```

**Pull request closed without merging**